UI/CardScrollUI.gd

```diff
@@ -0,0 +1,54 @@
+extends Control
+
+var card_scene: PackedScene = preload("res://Cards/Card.tscn")
+
+var discard_pile: Array[CardBase] = CardManager.card_container.discard_pile
+var draw_pile: Array[CardBase]    = CardManager.card_container.draw_pile
+var deck_pile: Array[CardBase]         = CardManager.card_container.default_deck
+var cards_to_display: Array[CardBase] = []
+var card_worlds: Array[CardWorld] = []
+var cardworld: CardWorld = null
+var cardui: Control = null
+
+const CARD_SCALE: Vector2 = Vector2(.7, .7)
+const CARDUI_INDEX: int = 2
+const CARDUI_POS: Vector2 = Vector2(4,4)
+const SIZE_OFFSET: Vector2 = Vector2(10,10)
+
+func _on_button_pressed() -> void:
+	queue_free()
+
+func _input(_inputevent: InputEvent) -> void:
```

Comment:
so that removes all the cards from the display when called right ?

Reply:
Yes, it deletes the root node CardScrollUI to be more specific.

Reply:
maybe a little comment would be good